foundations/foundation-common/src/test/java/io/servicecomb/foundation/common/part/TestAbstractPart.java

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Please update the license header with the new one.

Reply:
done